defender/middleware.py

```diff
@@ -18,9 +17,10 @@ def __init__(self, *args, **kwargs):
             # `LoginView` class-based view
             try:
                 from django.contrib.auth.views import LoginView
-                watch_login_method = method_decorator(watch_login)
+                our_decorator = watch_login()
+                watch_login_method = method_decorator(our_decorator)
                 LoginView.dispatch = watch_login_method(LoginView.dispatch)
             except ImportError:  # Django < 1.11
-                auth_views.login = watch_login(auth_views.login)
+                auth_views.login = watch_login()(auth_views.login)
```

Comment:
Is that correct? I'm no expert when it comes to decorators so might be correct, it just looks weird with `watch_login()(auth_views.login)`